openfisca_aotearoa/variables/acts/social_security/jobseeker_support/jobseeker_support.py

```diff
@@ -0,0 +1,246 @@
+#  calculating it depends on the applicable law at the requested `period`.
+#
+# For more information on OpenFisca's `variables`:
+# https://openfisca.org/doc/key-concepts/variables.html
+class jobseeker_support__entitled(variables.Variable):
+    value_type = bool
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Jobseeker Support eligibility and amount"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/whole.html#DLM6783144", "http://legislation.govt.nz/act/public/1964/0136/latest/DLM5478527.html"
+
+    # Old Job Seeker formula for 1964 Act, rewritten to show gaps
+    def formula_2013_04_17(persons, period, parameters):
+
+        # this whole section could be covered by a question "is the person seeking work, prepared, available and taking reasonable steps to find it"
+        ssa64_88B_1_a = numpy.logical_not(persons("social_security__full_employment", period))
+
+        # ssa64_88B_1_a_i is seeking work - not coded as could be covered by a general question
+        # ssa64_88B_1_a_ii is available - not coded as could be covered by a general question
+
+        ssa64_88B_1_a_iii = persons("jobseeker_support__willing_and_able", period)
+
+        ssa64_88B_1_a_iv = persons("jobseeker_support__taken_reasonable_steps", period)
+
+        ssa64_88B_1_a = ssa64_88B_1_a * ssa64_88B_1_a_iii * ssa64_88B_1_a_iv
+
+        # Exemption from Obligations: https://legislation.govt.nz/act/public/1964/0136/latest/DLM365312.html#DLM365312
+        # ssa64_88B_1_b TODO not in full time employment, would comply with a but also qualifies for exemption under section 105
+        ssa64_88B_1_c = numpy.logical_not(persons("social_security__full_employment", period)) * persons("jobseeker_support__limited_in_capacity", period)
+
+        ssa64_88B_1_d = (persons("social_security__employment", period) + persons("social_security__full_employment", period)) * persons("jobseeker_support__losing_earnings", period)
+
+        ssa64_88B_2 = persons("jobseeker_support__age_requirement", period)
+
+        ssa64_88B_3 = persons("social_security__residential_requirement", period.first_month)
+
+        ssa64_88B_4 = persons("jobseeker_support__minimum_income", period)
+
+        # ssa64_88B_5 temporary period makes income sufficient to fully abate benefit...
+
+        ssa64_88B_6 = persons("jobseeker_support__receiving", period) * \
+            persons("social_security__full_employment", period) * persons("jobseeker_support__full_employment_temporary", period) * \
+            persons("jobseeker_support__income_52_week_period_less_than", period)
+
+        # ssa64_88B_7 loss of earnings by payment to substitute due to sickness or injury
+
+        return ((ssa64_88B_1_a + ssa64_88B_6) + (ssa64_88B_1_a + ssa64_88B_1_c + ssa64_88B_1_d)) * ssa64_88B_2 * ssa64_88B_3 * (ssa64_88B_4 + ssa64_88B_6)
+
+    def formula_2018_11_26(persons, period, parameters):
```

Comment:
@asgharkazi Check this local variable naming to make matching with the Act more easily.
Also compare the complexity of the older 1964 act in the above formula with the better drafted 2018 ACT